src/config.ts

```diff
@@ -8,6 +8,10 @@ export function configurePWAOptions(options: ModuleOptions, nuxt: Nuxt) {
     options.outDir = publicDir ? resolve(publicDir) : resolve(nuxt.options.buildDir, '../.output/public')
```

Comment:
would be nice to get this from the resolved nitro config - which we can get from the `nitro:config` hook.

Reply:
seems nitro configuration not merged on `nitro:config`: not working on any build and in dev (`nitroConfig.output` is undefined)